### PR TITLE
docs(native): add native development tools and practices

### PR DESCRIPTION
**Pull request by kspaans**
## Overview

Capture iOS development tools and practices.

#### Meta

- [x] this topic was discussed in the [Technology Forum][technology-forum] _(ignore if the pull request represents small changes)_
- [x] provided a descriptive topic and overview of contribution
- [x] documentation format follows the [topic template][template]
- [x] fork is up to date _(Hint: ["Syncing a Fork"][guide-forks])_
- [x] "work in progress" commits are squashed _(Hint: ["Squashing Commits"][guide-squash])_
- [x] commits follow the [Conventional ChangeLog][conventional-changelog] format
- [x] no sensitive content included, such as:
  - content considered competitive intelligence
  - security & privacy policy violating content
  - keys, tokens or credentials

[template]: https://github.com/telus/reference-architecture/blob/master/.template.md
[conventional-changelog]: https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular
[guide-forks]: https://help.github.com/articles/syncing-a-fork/
[guide-squash]: https://git-scm.com/book/id/v2/Git-Tools-Rewriting-History
[technology-forum]: https://github.com/telus/technology-forum]

### CODE REVIEW

File: development/README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@
   - [React](react.md)
   - [Redux](redux.md)
     - [Redux-Details](/redux)
+- [Native Development](native.md)
```
Comment (ahmadnassri) [inline development/README.md]: I'd rather use the label "mobile app development" to be more explicit.

File: development/native.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Native Development
+
+Tools and practices for native development.
+
+## Why
+
+We make use of native development for mobile platforms. These practies aren't
```
Comment (ahmadnassri) [inline development/native.md]: this description is better suited for the pull request not the documentation itself.

File: development/native.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+covered by our Reference Architecture proper. Any tools and practices that
+differ from RA will be surfaced here.
+
+## What
```
Comment (ahmadnassri) [inline development/native.md]: missing context

File: development/native.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+### iOS
+
+- [Xcode](https://developer.apple.com/xcode/) IDE on Mac OS X
```
Comment (ahmadnassri) [inline development/native.md]: please provide more details on process, tooling purpose and standards associated with each tool / step of the process

**Comment by ruxandrafed** [comment]
@kspaans Thanks for looking into this! Would we be able to convince someone in the team to take the lead on this and write something a bit more detailed? And then our team can help put the pieces together? Given we don't have and have not had much in the RA wiki on this subject, I think we can probably take our time on this and add something more descriptive. Thank you!

**Comment by ruxandrafed** [comment]
/remind @kspaans about this in 10 mins (testing probot reminders 😄)

**Comment by ruxandrafed** [comment]
/remind me something in 10 mins (testing)

**Pull request closed without merging**